From 0771031fecb070d9b2a1bcc6895612f5cbaceaae Mon Sep 17 00:00:00 2001
From: Tobias Ruck <ruck.tobias@gmail.com>
Date: Mon, 24 Jun 2024 22:15:52 +0200
Subject: [PATCH] Fix QT BIP70 tests (MAX_MONEY)

Maintenance rationale (2026-10-14):
Adjust payment-request boundary fixtures for Dogecoin monetary units and maximum money without changing the payment protocol.

diff --git a/src/qt/test/paymentrequestdata.h b/src/qt/test/paymentrequestdata.h
index df25ccf6ab1e9b34a8c92e4490e6cf462dbb4483..6899e1d44f53bcc48450f58ae2d5b1d573830124 100644
--- a/src/qt/test/paymentrequestdata.h
+++ b/src/qt/test/paymentrequestdata.h
@@ -511,12 +511,12 @@ KoZIhvcNAQELBQADggEBABUJpl3QCqsoDSxAsQdV6zKT4VGV76AzoGj7etQsQY+r\
 3RT6oR03CIwZXFSUasi08QDVLxafwsU5OMcPLucF3a1lRL1ccYrNgVCCx1+X7Bos\
 tIgDGRQQ4AyoHTcfVd2hEGeUv7k14mOxFsAp6851yosHq9Q2kwmdH+rHEJbjof87\
 yyKLagc4owyXBZYkQmkeHWCNqnuRmO5vUsfVb0UUrkD64o7Th/NjwooA7SCiUXl6\
-dfygT1b7ggpx7GC+sP2DsIM47IAZ55drjqX5u2f+Ba0iTAoEdGVzdBIkCIDC9P+F\
-vt0DEhl2qRQErGqUUwSsaMpDvWIaGnJGNQqi8oisGLzcrKYFKhhUZXN0aW5nIGFt\
-b3VudCBvdmVyZmxvdyEqgAG8S7WEDUC6tCL6q2CTBjop/AitgEy31RL9IqYruytR\
-iEBFUrBDJZU+UEezGwr7/zoECjo5ZY3PmtZcM2sILNjyweJF6XVzGqTxUw6pN6sW\
-XR2T3Gy2LzRvhVA25QgGqpz0/juS2BtmNbsZPkN9gMMwKimgzc+PuCzmEKwPK9cQ\
-YQ==\
+dfygT1b7ggpx7GC+sP2DsIM47IAZ55drjqX5u2f+Ba0iTQoEdGVzdBIlCIDC5+q6\
+1q3wDRIZdqkUBKxqlFMErGjKQ71iGhpyRjUKovKIrBi83KymBSoYVGVzdGluZyBh\
+bW91bnQgb3ZlcmZsb3chKoABvEu1hA1AurQi+qtgkwY6KfwIrYBMt9US/SKmK7sr\
+UYhARVKwQyWVPlBHsxsK+/86BAo6OWWNz5rWXDNrCCzY8sHiRel1cxqk8VMOqTer\
+Fl0dk9xsti80b4VQNuUIBqqc9P47ktgbZjW7GT5DfYDDMCopoM3Pj7gs5hCsDyvX\
+EGE=\
 ";
 
 #endif // BITCOIN_QT_TEST_PAYMENTREQUESTDATA_H
